### PR TITLE
feat: Add multiple entry points with different tags for SEO

### PR DESCRIPTION
**Pull request by pswies**
Implement SEO additions demonstrated at https://test-fe.dydx.trade/ .

**Note:**
When moving to prod remove `"buildCommand"` from `vercel,json` and add the entry points generation to the build config in Vercel.

### CODE REVIEW

**Review by 2dpetkov — COMMENTED** [review]
It seems that I cannot build the project without invoking `generate-entry-points` 
I'm trying the current build command (without the new step) both with and without the `vite.config.ts` changes
```
pnpm run build --mode production && pnpm run build:inject-amplitude && pnpm run build:inject-bugsnag && pnpm run build:inject-intercom && pnpm run build:inject-statuspage && pnpm run build:inject-smartbanner
```

Is that expected? 
Can we also add the build steps to `README`?

**Comment by pswies** [comment]
@2dpetkov  Yes, this is expected. In `vite.config.ts` we have this code:

```
  build: {
    rollupOptions: {
      input: fs.readdirSync('entry-points').map(file => `/entry-points/${file}`)
    },
  },
```

This tells the application that all the entry points are expected in the `entry-points` directory.

In order to have this change merged but be able to retain the old functionality, we would have to ensure that the original `index.html` (currently renamed to `template.html`) exists in `entry-points`. Unfortunately I don't see a way to make the config in `vite.config.ts` optional, such that the entry points declaration gets overrided only if you run the script. I mean, we could modify this coonfig file dynamically but it would be rather unreadable.

Yes, I agree that a description in README is required. If we agree on the overall shape of this feature, I'll be happy to document everything. WDYT?

**Comment by 2dpetkov** [comment]
Gotcha!

So it seems like our `"build"` step will always require generating the entry points, how about change it to always do that?

**Comment by 2dpetkov** [comment]
on the `README` topic - absolutely, let's first agree on approach and then document

**Comment by pswies** [comment]
Good idea! I've moved the entry point generation to the `build` script. Now we were able to ditch the explicit build command in the config file.

File: vite.config.ts [hunk 1]
```diff
@@ -62,4 +63,9 @@ export default defineConfig(({ mode }) => ({
       '**/e2e/**',
     ],
   },
+  build: {
```
Comment (2dpetkov) [inline vite.config.ts:66]: What do we need this for?

Comment (2dpetkov) [reply vite.config.ts:66]: You answered above, sorry

**Comment by pswies** [comment]
I've added a description of this feature in README.

**Review by 2dpetkov — APPROVED** [review]

**Pull request closed without merging**